tests/test/statics.rs

```diff
@@ -0,0 +1,72 @@
+//! Tests for the static lifetime
+
+use super::*;
+
+
+#[test]
+fn static_lowering() {
+    lowering_success! {
+        program {
+            struct A<'a> where 'a: 'static {}
+            trait A<'a> where 'a: 'static {}
```

Comment:
Seeing a name collision here is somewhat surprising, even if it's allowed. Can the struct and trait have different names?

Reply:
Whoops, that's unintended.